### DISCUSSION TITLE
Welcome to ClassicAssist Discussions!

### DISCUSSION THREAD

**Discussion by Reetus** [Reetus/ClassicAssist#278] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 7FootSanta** [comment]
Hi. 7FootSanta here... I'm a. over 50 gamer that likes learning new things like python. Made a bunch of scripts for easyuo back in the day and I am looking forward to learning how to make them with classicassist. So when I ask a dumb question  just remember I am almost old.

**Comment by clintcorbinuo** [comment]
hey man. I am starting a server soon and I need to speak to you about providing me with a copy of classicassist with a couple of small added features. my email is clintcorbinuo@gmail.com

I'm not asking for anything for free, please email me as soon as you can!

Clint

**Comment by rinkanas** [comment]
Are you planning to do it for MacOS?
